extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -197,7 +200,8 @@ public function getDiscussionsWikiVariables() {
 				'discussionColorOverride' => SassUtil::sanitizeColor( $wgDiscussionColorOverride ),
 				'enableDiscussions' => $wgEnableDiscussions,
 				'enableDiscussionsImageUpload' => $wgEnableDiscussionsImageUpload,
-				'enableFeedsAndPosts' => $wgEnableFeedsAndPostsExt,
+				'enableFeedsAndPostsEmbeddedFeed' => $wgEnableFeedsAndPostsExt &&
```

Comment:
since you are renaming this variable, you should support both names for some time, because when mediawiki is released, mobile wiki will not be able to get value of this variable until it is released.

Reply:
Whatever, it's not on prod yet ;) In the worst scenerio embedded feeds won't be visible for some time.